### PR TITLE
chore: update dev container to better enable cpp and python liniting

### PR DESCRIPTION
**Pull request by apaletta3**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -27,11 +27,17 @@
...
+                "python.analysis.diagnosticSeverityOverrides": {
+                    "reportMissingImports": "none",
+                    "reportMissingModuleSource": "none",
+                    "reportUndefinedVariable": "none"
                 }
             },
             "extensions": [
                 "ms-vscode.cpptools",
                 "ms-python.python",
+                "ms-python.black-formatter",
```
Comment (Derollez) [inline .devcontainer/devcontainer.json:40]: Why do we remove this here? Black is used to format python files, shouldn't we have the associated extension?

Comment (apaletta3) [reply .devcontainer/devcontainer.json:40]: @Derollez I'm not removing it, I'm adding black to the dev container, it didn't exist before.

**Comment by apaletta3** [comment]
@vishwa2710 did you want to add anything linting/intellisense wise to this PR or should I land it?

**Review by phc1990 — APPROVED** [review]
LGTM

**Pull request merged**